src/sections/Project-grid/index.js

```diff
@@ -26,7 +26,7 @@ const ProjectPage = () => {
                             </Link>
                         </Col>
                         <Col xs={12} sm={4}>
-                            <Link to="#">  
+                            <Link to="/projects/service-mesh-performance">  
```

Comment:
Can't the projects be populated automatically as it is being done in the blog and news part?